Detectors/EMCAL/base/src/Digit.cxx

```diff
@@ -8,30 +8,45 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-#include <iostream>
 #include "EMCALBase/Digit.h"
+#include <TMath.h>
+#include <iostream>
 
 using namespace o2::EMCAL;
 
 ClassImp(Digit)
 
-Digit::Digit(Int_t module, Int_t tower, Double_t amplitude, Double_t time):
-FairTimeStamp(time),
-mModule(module),
-mTower(tower),
-mAmplitude(amplitude)
+  Digit::Digit(Int_t tower, Double_t amplitude, Double_t time)
+  : FairTimeStamp(time), mTower(tower), mAmplitude(amplitude)
+{
+}
+
+bool Digit::operator<(const Digit& other) const { return GetTimeStamp() < other.GetTimeStamp(); }
+
+bool Digit::operator>(const Digit& other) const { return GetTimeStamp() > other.GetTimeStamp(); }
```

Comment:
can you put these trivial functions as inline in the header?

Reply:
Sure